modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPackage.java

```diff
@@ -662,8 +644,15 @@ public void add(FeatureEntry entry, SimpleFeatureCollection collection) throws I
                     SimpleFeature f = it.next();
                     SimpleFeature g = w.next();
                     for (PropertyDescriptor pd : collection.getSchema().getDescriptors()) {
+                        /* geopkg spec requires booleans to be stored as SQLite integers this fixes
+                         * bug reported by https://osgeo-org.atlassian.net/browse/GEOT-5904 */
```

Comment:
The full link to the issue is unnecessary, just the ticket ID (GEOT-5904) is fine; eg:
> geopkg spec requires booleans to be stored as SQLite integers; this fixes GEOT-5904